getting_started/step_by_step/signals.rst

```diff
@@ -372,6 +503,15 @@ To emit values along with the signal, add them as extra arguments to the
         health -= amount
         emit_signal("health_changed", old_health, health)
 
+ .. code-tab:: csharp C#
+
+    public void TakeDamage(int amount)
+    {
+        var oldHealth = health;
+        Health -= amount;
+        EmitSignal("HealthChanged", oldHealth, Health);
```

Comment:
We recommend using `nameof` for the method name as it's less error-prone.

```suggestion
        EmitSignal(nameof(HealthDepleted), oldHealth, Health);
```

Reply:
done